server/views/pages/product.njk

```diff
@@ -77,6 +77,9 @@
           <li><a class="govuk-link--no-visited-state" href="/drift-radiator/products/{{ product.name | toMonitorName }}">
              Deployment drift
           </a></li>
+          <li><a class="govuk-link--no-visited-state" href="slack://channel?team=T02DYEB3A&id={{ product.slackChannelId | toMonitorName }}">
```

Comment:
do we need `toMonitorName` ?

Reply:
No, that's a nunjucks function, no need for that here.

Reply:
updated